Makefile

```diff
@@ -525,7 +524,7 @@ out/linters/golangci-lint-$(GOLINT_VERSION):
 ifeq ($(MINIKUBE_BUILD_IN_DOCKER),y)
 lint:
 	docker run --rm -v $(pwd):/app -w /app golangci/golangci-lint:$(GOLINT_VERSION) \
-	golangci-lint run ${GOLINT_OPTIONS} --skip-dirs "cmd/drivers/kvm|cmd/drivers/hyperkit|pkg/drivers/kvm|pkg/drivers/hyperkit" ./...
+	golangci-lint run ${GOLINT_OPTIONS} --config .golangci-build-in-docker.yaml ./...
```

Comment:
why do we have a different file for build in docker? can we use the same file for simplicity ?

Reply:
Since version 2 removed parameters like --skip-dirs , the same functionality can only be achieved through configuration files.
I personally prefer to use a single configuration file to reduce maintenance costs, and I'd like to spend some time to investigate further to see if there is a better way to solve this problem.

Reply:
The main difference between these two configuration files is that some directories are skipped